typings/vscode-mssql.d.ts

```diff
@@ -178,6 +178,22 @@ declare module 'vscode-mssql' {
 		osVersion: string;
 	}
 
+	/**
+	 * The possible values of the server engine edition
+	 */
+	export const enum DatabaseEngineEdition {
+		Unknown = 0,
+		Personal = 1,
+		Standard = 2,
+		Enterprise = 3,
+		Express = 4,
+		SqlDatabase = 5,
+		SqlDataWarehouse = 6,
```

Comment:
Should probably also update this to `SynapseAnalytics`

@shueybubbles Has there been any discussion over on the SMO side of updating the enums? I imagine not for compat reasons - maybe we should keep these the same here to match that.

Reply:
Actually let's just do that as a follow up task, you can go ahead and merge this in now and we can have a discussion about whether it makes sense to update later.

Reply:
I think if SMO were to do anything it would just add new symbols as synonyms for the current ones